packages/anthropic/src/anthropic-tools.ts

```diff
@@ -150,6 +198,87 @@ function textEditorTool_20241022<RESULT>(
   };
 }
 
+const TextEditor20250124Parameters = z.object({
```

Comment:
Same as above, [tool input schema](https://docs.anthropic.com/en/docs/agents-and-tools/computer-use#text-editor-tool) is the same for both versions.